src/Task/PhpCsFixerV2.php

```diff
@@ -91,6 +86,8 @@ public function run(ContextInterface $context)
             return $this->runOnAllFiles($context, $arguments);
         }
 
+        $arguments->add('--path-mode=intersection');
```

Comment:
@keradus, 

The only way we could know these classes is by adding php-cs-fixer to the dependencies, even for projects who don't want to use that tool. So this is something I would like to avoid. Or are you talking about some other way?

To be sure we are talking about the same thing: I was talking about a configuration inside grumphp named `intersect_on_commit` which will use the `intersect` path mode during a commit inside GrumPHP. There is no need for any changes in php-cs-fixer for that.